index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo">Techcolony</a>
+        <nav class="nav-bar">
+            <img class="drop-bar" src="/icons-images/Icon-menu.png">    
+        </nav>
```

Comment:
Nice job implementing the requirements of the previous reviewer👍 I can see you mistakenly introduced a bug in your toolbar, which is making it look different from the design😉 kindly see the images below and fix the issue😎

your project
<img width="317" alt="toolbar" src="https://user-images.githubusercontent.com/64643995/220896936-31348ab0-ebbb-4da8-af58-068ab8a35f7a.png">

design
![toolbar-mobile](https://user-images.githubusercontent.com/64643995/220896985-5fd11c25-9e19-4f9c-91a8-f8bb15bfddf8.png)

Reply:
@Ibizugbe @joel-kalema, I actually introduced it mistakenly but Joel didnt point it out. Its counting on my Review times.